main.lua

```diff
@@ -1400,7 +1404,7 @@ function EID:OnRender()
 
 			for _, entitySearch in ipairs(searchGroups) do
 				for _, entity in ipairs(entitySearch) do
-					if EID:hasDescription(entity) and entity.FrameCount > 0 and not EID.entitiesToPrint[GetPtrHash(entity)] then
+					if EID:hasDescription(entity) and (entity.FrameCount > 0 or entity:GetData()["EID_DescribeOnFirstFrame"]) and not EID.entitiesToPrint[GetPtrHash(entity)] then
```

Comment:
Use the EID:getEntityData(entity, str) function instead of directly calling entity:GetData(). The game is badly coded and sometimes forgets that GetData() should return a table, or return something at all...